### ISSUE TITLE
db:seed --class=blahblah has no response

### ISSUE THREAD

**Issue by z1haze** [laravel/framework#11375]
Hey, Just wanted to point out a tiny bug. When you run the database seeder and specify the seeder file with class=ClassNameHere you dont get any response back in the console such as Seeded: ClassNameHere

Nothing important, but I just wanted to share this with you in case you didnt know.

**Comment by GrahamCampbell** [comment]
I can't replicate. You should get an exception thrown.

**Comment by GrahamCampbell** [comment]
The code looks like this:

```

    /**
     * Get a seeder instance from the container.
     *
     * @return \Illuminate\Database\Seeder
     */
    protected function getSeeder()
    {
        $class = $this->laravel->make($this->input->getOption('class'));

        return $class->setContainer($this->laravel)->setCommand($this);
    }
```

**Comment by alexhouse** [comment]
I think the issue is if you specify a class to seed (that exists), artisan doesn't provide any confirmation output (unlike just calling `db:seed` with no arguments where you get information on the classes that get seeded

**Comment by GrahamCampbell** [comment]
I think that's right?

**Comment by alexhouse** [comment]
Technically, yes. Perhaps there's risk of confusion, though, as calling `db:seed` confirms every seeder that runs, but calling an individual one doesn't necessarily output that execution is successful, it just errors if something goes wrong.

**Comment by z1haze** [comment]
Ok well whether it's a bug or not I just think it should provide output of success. Every other artisan function I've used seems to provide some sort of feedback

**Comment by MaurizioBrioschi** [comment]
Just FYI, this is not solved also in the last Laravel Version (5.6)

**Comment by z1haze** [comment]
Yes, all I ever had mentioned was there was no success feedback after running db:seed and specifying a seeder, but you get success feedback when you call db:seed without any arguments. I believe it should be consistent.

**Comment by z1haze** [comment]
Great, only took 3 years!
